packages/cli/src/zosfiles/compare/ds/Dataset.handler.ts

```diff
@@ -70,43 +76,116 @@ export default class DatasetHandler extends ZosFilesBaseHandler {
         let dsContentString1 = "";
         let dsContentString2 = "";
 
-        if(commandParameters.arguments.noseqnum){
+        if (commandParameters.arguments.noseqnum) {
             const seqnumlen = 8;
 
             const dsStringArray1 = dsContentBuf1.toString().split("\n");
-            for(const i in dsStringArray1){
+            for (const i in dsStringArray1) {
                 const sl = dsStringArray1[i].length;
-                const tempString = dsStringArray1[i].substring(0,sl - seqnumlen);
+                const tempString = dsStringArray1[i].substring(0, sl - seqnumlen);
                 dsContentString1 += tempString + "\n";
             }
 
             const dsStringArray2 = dsContentBuf2.toString().split("\n");
-            for(const i in dsStringArray2){
+            for (const i in dsStringArray2) {
                 const sl = dsStringArray2[i].length;
-                const tempString = dsStringArray2[i].substring(0,sl - seqnumlen);
+                const tempString = dsStringArray2[i].substring(0, sl - seqnumlen);
                 dsContentString2 += tempString + "\n";
             }
         }
-        else
-        {
+        else {
             dsContentString1 = dsContentBuf1.toString();
             dsContentString2 = dsContentBuf2.toString();
         }
 
         let jsonDiff = "";
         let expandflag = true;
         const contextLinesArg = commandParameters.arguments.contextlines;
-        if(contextLinesArg >= 0){
+        if (contextLinesArg >= 0) {
             expandflag = false;
         }
-        jsonDiff = diff(dsContentString1, dsContentString2, {aAnnotation: "Removed",
+
+        jsonDiff = await diff(dsContentString1, dsContentString2, {
+            aAnnotation: "Removed",
             bAnnotation: "Added",
             aColor: TextUtils.chalk.red,
             bColor: TextUtils.chalk.green,
             contextLines: contextLinesArg,
             expand: expandflag
         });
 
+
+        //  CHECHKING IIF THE BROWSER VIEW IS TRUE, OPEN UP THE DIFFS IN BROWSER
+        if (browserView) {
+            const patchDiff = createTwoFilesPatch(
+                commandParameters.arguments.dataSetName1, commandParameters.arguments.dataSetName2, dsContentString1, dsContentString2
+            );
+
+            const diffHtml = html(patchDiff, {
+                // drawFileList: true,
+                outputFormat: "side-by-side",
+                matching: "lines",
+                diffStyle: "char",
+            });
+
+            const diffLauncher = path.join(ImperativeConfig.instance.cliHome, './diff.html');
```

Comment:
The diff HTML file appears to be created right under cliHome. Should it be placed into a subdirectory? The diff file never appears to be deleted. Would this create a lot of clutter over time?

Reply:
Good observation! 😋 
There are plans to migrate all of these HTML structure to the Imperative framework.
And similar to the web-help, we will attempt to reuse most of the structure and styling. 😉

Reply:
Hey @gejohnston, 
Sorry for the delay.

Actually, we are working on the same thing to migrate the getting diff strings and opening diff in-browser utility to the imperative framework itself as @zFernand0 mentioned. 

Please do refer to [Imperative -  Diff utilities](https://github.com/zowe/imperative/tree/diff-utilities) for the same. 
And in the same regard, we will be having a web-diff dir at the cli home, and the web-diff in browser utility will be managed there only.

Thanks!